backend/src/test/java/com/woowacourse/kkogkkog/application/CouponServiceTest.java

```diff
@@ -25,6 +25,8 @@
 import org.springframework.transaction.annotation.Transactional;
 
 @Transactional
+@Nested
+@DisplayName("CouponService 의")
 public class CouponServiceTest extends ServiceTest {
```

Comment:
`"CouponService 의"`를 "CouponService의" 혹은 "CouponService 클래스의"로 수정하는 것은 어떠할까요? 
둘을 합치면 인텔리제이에서 오타로 인식해서 띄어쓴 것으로 보이는데, 읽을 때 좀 어색하네요.

```suggestion
@Nested
@DisplayName("CouponService 클래스의")
public class CouponServiceTest extends ServiceTest {
```

Reply:
기존에 `클래스의`를 썼었는데 이번에 문구가 길어보여서 빼봤습니다.
추가하는 것도 저는 괜찮습니다!